contrib/ay-openqa-worker-leap.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+    </mode>
+  </general>
+  <networking>
+    <!-- For multiple network interfaces by default the last is configured, prefer the first -->
+    <interfaces config:type="list">
+      <interface>
+        <bootproto>dhcp</bootproto>
+        <name>eth0</name>
+        <startmode>auto</startmode>
+      </interface>
+    </interfaces>
+    <keep_install_network config:type="boolean">true</keep_install_network>
+  </networking>
+  <software>
+    <!-- It seems unfortunately we need to explicitly specify a product, would have preferred a more generic config -->
+    <products config:type="list">
+      <product>Leap</product>
+    </products>
+    <packages config:type="list">
+      <package>openssh</package>
+      <package>sudo</package>
+      <!-- could not find salt-minion in default Leap repos, disabling, handling in 2nd stage script -->
+      <!--<package>salt-minion</package>-->
+    </packages>
+  </software>
+  <partitioning config:type="list">
+    <drive>
+      <initialize config:type="boolean">true</initialize>
+      <partitions config:type="list">
+        <partition>
+          <mount>/</mount>
+          <size>max</size>
+          <filesystem config:type="symbol">btrfs</filesystem>
+        </partition>
+        <partition>
+          <mount>swap</mount>
+          <size>auto</size>
+        </partition>
+      </partitions>
+    </drive>
+  </partitioning>
+  <scripts>
+    <post-scripts config:type="list">
+      <script>
+        <filename>setup.sh</filename>
+        <interpreter>shell</interpreter>
+        <debug config:type="boolean">true</debug>
+        <source><![CDATA[
+            timeout 600 sh -c 'until ! pgrep -a zypper; do sleep 5; done'
+            zypper -n in salt-minion
+            echo 'roles: worker' > /etc/salt/grains
+            systemctl enable --now sshd salt-minion
```

Comment:
Maybe add a comment that this doesn't work at this point reliably and one might invoke those commands manually instead. It is also useless to start salt-minion without configuring the salt master.